core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2363,4 +2363,12 @@ package object config {
       .version("3.3.0")
       .intConf
       .createWithDefault(5)
+
+  val SPARK_PROPERTIES_KEY_QUEUE = ConfigBuilder("spark.properties.key.queue")
+    .doc("Specifies the spark property key name for queue information." +
+      " When it is specified, the eventLog parser could use it as the key" +
+      " to extract the queue value from the Spark Properties in the eventLog files.")
+    .version("3.3.0")
```

Comment:
3.4.0?

Reply:
Yes, good catch, and will change it. Thanks.